gatsby-config.js

```diff
@@ -29,12 +29,26 @@ module.exports = {
     `gatsby-transformer-remark`,
     `gatsby-plugin-styled-components`,
     `gatsby-plugin-sitemap`,
-    {
-      resolve: 'gatsby-plugin-segment-js',
-      options: {
-        prodKey: 'JXEYLKE1T9ptsDlNqeNIMdoOy1Ept8CB',
-        trackPage: true,
-      },
-    },
+    ...(process.env.GOOGLE_ANALYTICS_TRACKING_ID && !isDeployPreview
+      ? [
+          {
+            resolve: 'gatsby-plugin-google-analytics',
+            options: {
+              trackingId: process.env.GOOGLE_ANALYTICS_TRACKING_ID,
+              head: true,
+            },
+          },
+        ]
+      : []),
+    ...(process.env.FACEBOOK_PIXEL_ID && !isDeployPreview
+      ? [
+          {
+            resolve: 'gatsby-plugin-facebook-pixel',
+            options: {
+              pixelId: process.env.FACEBOOK_PIXEL_ID,
+            },
+          },
+        ]
+      : []),
```

Comment:
I pulled the env vars into Netlify. No reason to check them in here, I think. I also validated that they work by altering this logic above and removing `!isDeployPreview` to make sure the pixels were there with the right IDs. Now that I know they work, I don't think we should be tracking users on the deploy previews -- only on prod.